include/z64actor_dlftbls.h

```diff
@@ -0,0 +1,51 @@
+#ifndef Z64ACTOR_DLFTBLS_H
+#define Z64ACTOR_DLFTBLS_H
+
+#include "PR/ultratypes.h"
+#include "libc/stdint.h"
+#include "segment_symbols.h"
+
+// This value is hardcoded to be the size of ovl_Arrow_Fire which currently is the biggest actor that uses the AM_FIELD.
+DECLARE_OVERLAY_SEGMENT(Arrow_Fire)
+#define AM_FIELD_SIZE SEGMENT_SIZE(ovl_Arrow_Fire)
+
+struct ActorInit;
+
+#define DEFINE_ACTOR(_name, enumValue, _allocType, _debugName) enumValue,
+#define DEFINE_ACTOR_INTERNAL(_name, enumValue, _allocType, _debugName) enumValue,
+#define DEFINE_ACTOR_UNSET(enumValue) enumValue,
+
+typedef enum ActorId {
+    #include "tables/actor_table.h"
+    /* 0x2B2 */ ACTOR_ID_MAX // originally "ACTOR_DLF_MAX"
+} ActorId;
+
+#undef DEFINE_ACTOR
+#undef DEFINE_ACTOR_INTERNAL
+#undef DEFINE_ACTOR_UNSET
+
+typedef enum AllocType {
+    /* 0 */ ALLOCTYPE_NORMAL,
+    /* 1 */ ALLOCTYPE_ABSOLUTE,
+    /* 2 */ ALLOCTYPE_PERMANENT
+} AllocType;
+
+typedef struct ActorOverlay {
+    /* 0x00 */ uintptr_t vromStart;
+    /* 0x04 */ uintptr_t vromEnd;
```

Comment:
Definitely not for this PR (and probably should be discussed with OoT as well) but a part of me wonder if for overlay's like this, we should actually use `RomFile` for the `vromStart` and `vromEnd`, i.e. this would be like 
```c
/* 0x0 */ RomFile rom;
/* 0x8 */ void* vramStart;
...
```
(or some other name not sure). This would let us use the `ROM_FILE` macros a bit more.

Reply:
Note I do plan on doing this (like https://github.com/zeldaret/oot/pull/1742) after this PR gets merged.